src/test/arbitrum/ArbitrumStEth.t.sol

```diff
@@ -0,0 +1,266 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+import {AaveV3Arbitrum} from 'aave-address-book/AaveAddressBook.sol';
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {ProtocolV3TestBase, ReserveConfig, ReserveTokens, IERC20} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {BridgeExecutorHelpers} from 'aave-helpers/BridgeExecutorHelpers.sol';
+import {ProtocolV3TestBase, ReserveConfig} from 'aave-helpers/ProtocolV3TestBase.sol';
+import {AaveGovernanceV2, IExecutorWithTimelock} from 'aave-address-book/AaveGovernanceV2.sol';
+import {CrosschainForwarderArbitrum} from '../../contracts/arbitrum/CrosschainForwarderArbitrum.sol';
+import {AddressAliasHelper} from '../../contracts/arbitrum/AddressAliasHelper.sol';
+import {StEthPayload} from '../../contracts/arbitrum/StEthPayload.sol';
+import {DeployL1ArbitrumProposal} from '../../../script/DeployL1ArbitrumProposal.s.sol';
+import {IInbox} from '../../interfaces/arbitrum/IInbox.sol';
+import {IL2BridgeExecutor} from '../../interfaces/IL2BridgeExecutor.sol';
+
+contract ArbitrumStEthE2ETest is ProtocolV3TestBase {
+  // the identifiers of the forks
+  uint256 mainnetFork;
+  uint256 arbitrumFork;
+
+  StEthPayload public stEthPayload;
+
+  address public constant INBOX_ADDRESS =
```

Comment:
Would get this from `CrosschainForwarderArbitrum` this way the tests rely on what will be deployed